script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1391,109 +1586,133 @@ I don't give a damn whether she was being serious or just trying to rile me up.
+It's clear that either way,
+ Senpai thinks Arcueid is nothing more than a vampire to be defeated,
+ and that I'm a fool for trusting her.
 }
 [sha:49995e4ab7f9806145ef55e1a2452aac7ad17d18]{
 -- Page 253, Offset 22787.
 -- 「俺は騙されてなんかないっ！　だいたい、アイツが嘘なんかつけるもんか、先輩みたいに器用じゃないんだから！
-"I'm not being deceived! Arcueid doesn't lie anyway, she isn't cunning like you!
+// 2 dumb 2 lie
+"I'm not being deceived!
+ As if she's even capable of lying in the first place.
+ She's not a snake like you, Senpai!
 }
 [sha:42c436afdaf915d0481278b6534ff4e9d1aa27c5]{
 -- Page 253, Offset 22788.
 -- 俺はアイツを信じます。っていうか、アレが嘘でも喜んで騙されてやる！」
 // Ciel big L in her own route
-I trust her. Scrap that, even if she's lying, I'll gladly be deceived!"
+I believe what she said.
+ Fuck it, even if she %{i}is%{/i} lying, I'll gladly be fooled by her!"
 }
 [sha:5ad664558d4e40872b8debd606e43987ac1043bf]{
 -- Page 254, Offset 22789.
 -- 「あのですね。自分が何を言っているのか理解していますか？　社会を守る私たちより、自由気ままに動いては街を混乱に陥れる、あのはた迷惑な真祖を信じるって言ったんですよ？
-"Excuse me? Do you even understand what you're saying? Did you just tell me that you'd believe that irksome True Ancestor who plunged this city into chaos on a whim over people like us who are protecting society?
+"Excuse me?
+ Can you even hear yourself right now?
+ She plunged this city into chaos on a whim.
+ Did you seriously just tell me that you believe that devil-may-care True Ancestor over us,
+ the people who are actually working to %{i}protect%{/i} society?
 }
 [sha:0fe781b7dde00087e043c2a08653540eca0b99b4]{
 -- Page 254, Offset 22790.
 -- さすがにそれは聞き逃せません。っていうか、わたしをバカにしてるんですか!?」
-Hoh, I'm sure I must have misheard. Are you making fun of me?!"
+I'm sure I must have simply misheard you.
+ Or are you actively trying to make fun of me?!"
 }
 [sha:c79dbe32287e9cec99c431b7f934aabd8847686b]{
 -- Page 255, Offset 22791.
 -- 「バカにしてるんじゃなくて、アルクェイドを信じるって言ってるだけでしょう!?
-"I'm not making fun of you, I'm simply telling you that I trust Arcueid!
+"I'm not making fun of you, all I'm saying is that I trust Arcueid!
 }
 [sha:7a6a0429c1c1b29cfa87db51653ce215621c86b3]{
 -- Page 255, Offset 22792.
 -- 先輩こそどうして分かってくれないんですか!?　アイツと協力すれば、街はすぐに元通りになるじゃないですか！」
-Why don't you, of all people, understand me?! If you two worked together, the city would immediately be back to normal!"
+Why don't you, of all people, understand me?!
+ If you two worked together, the city would be back to normal in no time!"
 }
 [sha:046eb5041853658dc403cbc01c9ac4ad58eef29f]{
 -- Page 256, Offset 22793.
 -- 「彼女と協力するのなんて死んでもお断りです。月を重しにして眠った方がまだマシですから」
 // A throwback to file 05_01_CIEL05_3D line 271 where Arcueid says あいつに手を貸すぐらいなら、太陽を抱いて眠ってもいいぐらい (I'd rather breathe my last while embracing the sun than help her)
-"I'd rather die than cooperate with her. I'd rather breathe my last with the moon pressing down on me."
+"I'd rather die than cooperate with her.
+ I'd let the moon crush the life out of me before I stooped to that."
```

Comment:
Both of these statements are kinda funny, I think they should be closer to JP and more alike one-another.
>I'd rather sleep in the bosom of the sun than...
>I'd rather sleep under the weight of the moon than...

Makes this this one more like
>You'd have to kill me before I cooperate with her. 
>I'd rather sleep under the weight of the moon before stooping to that.

"Yeah, but doesn't that kinda sound weird?" Yeah, it is weird. Weird enough that Shiki comments on it both times.